README.md

```diff
@@ -52,6 +52,18 @@ Variables:
 - `datadog_agent5` - install agent5 instead of agent6 (default to `false`)
 - `datadog_agent5_apt_repo` - Override default Datadog `apt` repository for agent5
 
+To enable the process agent on agent 5, you need to set on `datadog_config`:
```

Comment:
Should we create a specific section in the README for the process agent ? It could make thing simpler/clearer.
What do you think ?

Reply:
I agree with you. It'll make the README file clearer. I've just pushed a new commit with the suggested modification